parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -239,7 +243,7 @@ pub mod pallet {
 
 	impl<T: Config> Pallet<T> {
 		/// Generate a messages commitment and insert it into the header digest
-		pub(crate) fn commit_messages() {
+		pub(crate) fn do_commit_messages() {
```

Comment:
This name change is unnecessary.

I only named `do_process_message` that way so as not to clash with the existing `process_message`.

Reply:
[Rename to commit_messages](https://github.com/Snowfork/snowbridge/pull/975/commits/6691413ab99ce893189b3409249541084141c42f)